Hugo Cullinan/week01/day-3/js-homework-1.js

```diff
@@ -0,0 +1,84 @@
+// 1. Define a function `maxOfTwoNumbers` that takes two numbers as arguments and returns the largest of them. Use the if-then-else construct available in Javascript. You'll have to remember your pre-work, or do some googling to figure this out.
+const maxOfTwoNumbers = function (a, b) {
+  if (a > b) {
+    return a;
+  }else {
```

Comment:
Generally its preferred to add an extra space here; i.e.   `} else {`
Very minor feedback, but writing code that has a nice layout makes it soooooo much easier to read when you create bigger programs.